File: Modules/Common/LES/SIE.Web.LES/StockOrders/Commands/StockOrderMergeIssuedEditCommand.cs
using SIE.Web.Command;

namespace SIE.Web.LES.StockOrders.Commands
{
    public class StockOrderMergeIssuedEditCommand : ViewCommand
    {
        protected override object Excute(ViewArgs args, string scope)
        {
            return true;
        }
    }
}

File: Modules/Common/LES/SIE.Web.LES/StockOrders/Commands/StockOrderMergeIssuedEditCommand.js
SIE.defineCommand('SIE.Web.LES.StockOrders.Commands.StockOrderMergeIssuedEditCommand', {
    extend: 'SIE.cmd.Edit',
    meta: { text: "修改", group: "edit", iconCls: "icon-EditEntity icon-blue" },
    canExecute: function (view) {
        var current = view.getCurrent();
        if (current) {
            if (current.getState() == 0)
                return true;
        }
        return false;
    },
    onEditting: function (entity) {
        if (entity) {
            entity.setState(0);
        }
    },
});